src/qibojit/custom_operators/ops.py

```diff
@@ -90,3 +90,29 @@ def measure_frequencies(frequencies, probs, nshots, nqubits, seed, nthreads):
             frequencies_private[shot] += 1
     frequencies += thread_frequencies.sum(axis=0)
     return frequencies
+
+
+@njit(cache=True, parallel=True)
+def transpose_state(pieces, state, nqubits, order):
```

Comment:
I am getting a deprecation warning for `pieces` and `order`, I believe they should be converted to `List`, accordingly to the numba docs: https://numba.pydata.org/numba-doc/latest/reference/deprecation.html#deprecation-of-reflection-for-list-and-set-types

Reply:
Since both `pieces` and `order` are not modified by this function I cast them to `tuple` which is immutable and does not have these issues. This should fix the warning. Let me know if you agree.

Reply:
Perfect, thanks.